collectors/ps_constants/core.py

```diff
@@ -88,3 +95,15 @@ def sync_special_consideration_packages(sc_packages):
     for name in sc_packages:
         package = SpecialConsiderationPackage(name=name)
         package.save()
+
+
+@transaction.atomic
+def sync_sla_policies(sla_policies):
+    """
+    Sync SLA policy data
+    """
+    SLAPolicy.objects.all().delete()
```

Comment:
What happens if ps_constants become unavailable? Will all SLA policies be deleted? I realize this is in principle a part you didn't change, but I think this now became `ps_constants_collector`'s biggest responsibility that would pose a bigger issue if the state was deleted than the constants before it.

Reply:
The data is fetched by `fetch_ps_constants`, and if there is an error either in the request or loading the YAML, this function will raise an error and the `ps_constants_collector` will not continue.